rpm/389-ds-base.spec.in

```diff
@@ -540,11 +582,11 @@ if ! make DESTDIR="$RPM_BUILD_ROOT" check; then cat ./test-suite.log && false; f
 
 %post
 if [ -n "$DEBUGPOSTTRANS" ] ; then
-    output=$DEBUGPOSTTRANS
```

Comment:
Indentation was removed here and below, is this intentional?

Reply:
Good catch!